subprojects/packagefiles/libgpiod/readme.txt

```diff
@@ -0,0 +1,3 @@
+This repository contains a Meson build definition for project libgpiod.
```

Comment:
This file is no longer needed and doesn't make much sense inside the subprojects setup, IMO.

Reply:
Right, I put it there because I was running `mesonwrap review` against the separate repo I had going. This was before it was pointed out to me that the process had changed.